src/learn/zkcloud/snarkos_build_guide.md

```diff
@@ -0,0 +1,187 @@
+# How To set up SnarkOS
+
+### <a name='TableofContents'></a>Table of Contents
+
+* [1. Build Guide](#1-build-guide)
+  * [1.1 Requirements](#11-requirements)
+  * [1.2 Installation](#12-installation)
+* [2. Run an Aleo Node](#2-run-an-aleo-node)
+  * [2.1 Run an Aleo Client](#21-run-an-aleo-client)
+  * [2.2 Run an Aleo Prover](#22-run-an-aleo-prover)
+* [3. Command Line Interface](#3-command-line-interface)
+* [4. FAQs](#4-faqs)
+## 1. Build Guide
+### 1.1 Requirements
+
+The following are **minimum** requirements to run an Aleo node:
```

Comment:
How are we going to track changes in the original documentation when this requirements change?

Reply:
I think best have someone clearly designate to update the requirements if and when they change.